onnxruntime/contrib_ops/cuda/bert/attention_impl.cu

```diff
@@ -465,6 +465,23 @@ Status UnfusedAttention(
   DUMP_TENSOR_D("K", data.k, batch_size, num_heads, qk_head_size, sequence_length);
   DUMP_TENSOR_D("QK", data.scratch, batch_size, num_heads, sequence_length, total_sequence_length);
 
+  if (data.positional_embedding != nullptr){
```

Comment:
Like rotary embedding, this positional embedding logic shall be in a separate operator. We do not want to make MHA too complex.